catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/content/brick/BrickTestUtils.java

```diff
@@ -68,19 +68,20 @@ public static void checkIfBrickAtPositionShowsString(int position, String string
 				.check(matches(isDisplayed()));
 	}
 
-	public static void checkIfSpinnerOnBrickAtPositionShowsString(int spinnerResourceId, int position, int
-			stringResourceId) {
+	public static void clickAndSelectFromSpinnerOnBrickAndCheckIfSpinnerShowsString(int spinnerResourceId, int
+			position, int stringResourceId) {
+		onScriptList().atPosition(position).onChildView(withId(spinnerResourceId))
+				.perform(click());
+		onData(allOf(is(instanceOf(String.class)), is(UiTestUtils.getResourcesString(stringResourceId))))
+				.perform(click());
 		onScriptList().atPosition(position).onChildView(withId(spinnerResourceId))
```

Comment:
please use call to checkIfSpinnerOnBrickAtPositionShowsString instead of "manually" going there.